src/core/BufferGeometry.js

```diff
@@ -879,6 +879,14 @@ BufferGeometry.prototype = Object.assign( Object.create( EventDispatcher.prototy
 
 	},
 
+	hasColorAlpha: function () {
```

Comment:
I don't think we want to expose this method.

Reply:
Yes, this also looks strange to me. Any suggestions where I should put this method? Don't feel like to repeat the code in `WebGLRenderer.js` and `WebGLPrograms.js`.

Reply:
Agreed! I was going to clean that up after merging it tomorrow.

Reply:
> I would indeed inline it for now.

Sure thing. Updated.